sympy/printing/repr.py

```diff
@@ -180,6 +180,9 @@ def _print_AppliedPredicate(self, expr):
 
     def _print_str(self, expr):
         return repr(expr)
+    # XXX: Can be removed once Py2 support is dropped
+    def _print_basestring(self, expr):
+        return repr(expr)
```

Comment:
@asmeurer Since SymPy 1.5 will be the last version supporting Python 2.x, so probably this PR can be resolved. @ritesh99rakesh What do you say?